public/locales/en-US/translations.json

```diff
@@ -160,6 +162,11 @@
   "transaction_EnableAmendment": "Enable Amendment",
   "transaction_SetFee": "Set Fee",
   "transaction_UNLModify": "UNL Modify",
+  "dex_transaction": "Dex transaction",
```

Comment:
These should all be plural and consistent with capitalization

Reply:
done